Jenkinsfile

```diff
@@ -0,0 +1,497 @@
+                        env.playbook = "ci-pipeline/config/duffy-setup/setup-rpmbuild-system.yml"
+                        env.ref = "fedora/${fed_branch}/${basearch}/atomic-host"
+                        env.repo = "${env.fed_repo}"
+                        env.rev = "${env.fed_rev}"
+                        env.basearch = "x86_64"
+                        env.ANSIBLE_HOST_KEY_CHECKING = "False"
+                        env.DUFFY_OP = "--allocate"
+
+                        // Send message on fedmsg
+                        sendCIMessage messageContent: '',
+                                messageProperties: "topic=${topic}\n" +
+                                        "build_url=${BUILD_URL}\n" +
+                                        "build_id=${BUILD_ID}\n" +
+                                        "branch=${branch}\n" +
+                                        "ref=fedora/${branch}/${basearch}/atomic-host\n" +
+                                        "rev=${fed_rev}\n" +
+                                        "repo=${fed_repo}\n" +
+                                        "namespace=${namespace}\n" +
+                                        "username=fedora-atomic\n" +
+                                        "test_guidance=''\n" +
+                                        "status=success",
+
+                                messageType: 'Custom',
+                                overrides: [topic: "${topic}"],
+                                providerName: 'fedora-fedmsg'
+
+                        // Provision of resources
+                        allocDuffy("${current_stage}")
+
+                        echo "Duffy Allocate ran for stage ${current_stage} with option ${env.DUFFY_OP}\r\n" +
+                                "ORIGIN_WORKSPACE=${env.ORIGIN_WORKSPACE}\r\n" +
+                                "ORIGIN_BUILD_TAG=${env.ORIGIN_BUILD_TAG}\r\n" +
+                                "ORIGIN_CLASS=${env.ORIGIN_CLASS}"
+                        def job_props = "${env.ORIGIN_WORKSPACE}/job.props"
+                        def job_props_groovy = "${env.ORIGIN_WORKSPACE}/job.groovy"
+                        convertProps(job_props, job_props_groovy)
+                        load(job_props_groovy)
+
+                        // Stage resources - RPM build system
+                        setupStage()
+
+                        // Rsync Data
+                        writeFile file: "${env.ORIGIN_WORKSPACE}/task.env",
+                                  text: "export JENKINS_JOB_NAME=\"${JOB_NAME}-${current_stage}\"\n" +
+                                        "export JENKINS_BUILD_TAG=\"${BUILD_TAG}-${current_stage}\"\n" +
+                                        "export OSTREE_BRANCH=\"\${OSTREE_BRANCH:-}\"\n" +
+                                        "export fed_repo=\"${fed_repo}\"\n" +
+                                        "export fed_branch=\"${fed_branch}\"\n" +
```

Comment:
In the current workflow, if fed_branch = rawhide we create the new variable branch which points to master.  The reason why we create that branch variable instead of simply overwriting fed_branch is because for example the rpmbuild uses "master" instead of rawhide.  If we did it properly, for f27 right now, branch should be rawhide and fed_branch should be master throughout to the best of my knowledge